### PR TITLE
Update sv.js

### PR DESCRIPTION
**Pull request by wipcore-jonasboman**
A complete translation to Swedish.

### CODE REVIEW

File: locales/sv.js [hunk 1]
```diff
@@ -1,316 +1,632 @@
...
+promo_code_code:
+"Har du en kampanjkod?",
+promo_code_rate_on_order:
+"Rabatt på din order",
+promo_code_alternate_price:
+"Specialpris på produkter",
+total:
+"Totalt",
+total_paid:
+"Total paid",
```
Comment (thatfrankdev) [inline locales/sv.js]: Is that really the correct form?

File: locales/sv.js [hunk 2]
```diff
@@ -1,316 +1,632 @@
...
+shipping_address:
+"Leveransadress",
+payment_informations:
+"Betalningsinformation",
+payment_informations_bill_me_later:
+"Jag vill betala mot faktura",
+payment_informations_no_payment_required:
+"Ingen betalning krävs",
+payment_informations_paypalexpress:
+"Betala med Paypal",
```
Comment (thatfrankdev) [inline locales/sv.js]:
```suggestion
"Betala med PayPal",
```

File: locales/sv.js [hunk 3]
```diff
@@ -1,316 +1,632 @@
...
+payment_method_card_holder:
+"Namn på kortet",
+payment_method_card_type:
+"Korttyp",
+payment_method_card_number:
+"Kortnummer",
+payment_method_card_cvc:
+"CVC",
+payment_method_card_exp_month:
+"Giltigt till månad",
```
Comment (thatfrankdev) [inline locales/sv.js]: Why remove year?

File: locales/sv.js [hunk 4]
```diff
@@ -1,316 +1,632 @@
...
+subscription_cancel_confirmation:
+"Vill du verkligen avbryta ditt abonnemang?",
+subscription_cancel_success:
+"Ditt abonnemang har avslutats.",
+subscription_cancelled_notice:
+"Abonnemanget är inte längre aktivt, den avbröts {0}.",
+subscription_cancelledon:
+"Avbröts",
+paypal_express_checkout_link:
+"Betala med Paypal",
```
Comment (thatfrankdev) [inline locales/sv.js]:
```suggestion
"Betala med PayPal",
```

File: locales/sv.js [hunk 5]
```diff
@@ -1,316 +1,632 @@
...
+paypal_express_checkout_title:
+"Betala med Paypal",
```
Comment (thatfrankdev) [inline locales/sv.js]:
```suggestion
"Betala med PayPal",
```

File: locales/sv.js [hunk 6]
```diff
@@ -1,316 +1,632 @@
...
+item_invalid_must_remove:
+"Set ser ut som att artikeln inte längre är tillgänglig. Du behöver eventuellt kontakta webbplatsägaren för att lösa detta då det kan vara ett fel i konfigurationen. När du går vidare kommer artikeln att tas bort från kundvagnen.",
+accept_cart_changes:
+"Acceptera ändringarna och fortsätt",
+payment_failed_text:
+"Ursäkta, vi har inte kunnat genomföra betalningen. Du kan fortsätta handla eller försöka igen genom att trycka på knapparna nedan.",
+payment_method_willbepaidlater:
+"Faktura",
+payment_method_paypal:
+"Paypal",
```
Comment (thatfrankdev) [inline locales/sv.js]:
```suggestion
"PayPal",
```

File: locales/sv.js (hunk 1 above)
Comment (wipcore-jonasboman) [reply locales/sv.js]: Nope, will update. Thanks

File: locales/sv.js (hunk 3 above)
Comment (wipcore-jonasboman) [reply locales/sv.js]: payment_method_card_exp_year exist in file

**Comment by wipcore-jonasboman** [comment]
@thatfrankdev thanks for the review. Have made some justifications

File: locales/sv.js (hunk 3 above)
Comment (thatfrankdev) [reply locales/sv.js]: I missed that. All good, thanks

**Pull request merged**